overview/templates/overview/candidates_basic.html

```diff
@@ -0,0 +1,167 @@
+                                    </div>
+                                </td>
+
+                                <td>
+                                    {% if candidate.is_incumbent %}
+                                        <i class="fa fa-check text-success"></i>
+                                    {% endif %}
+                                </td>
+
+                                <td>
+                                    {{ candidate.short_history_text }}
+                                </td>
+
+                                <td>
+                                    {{ candidate.political_party }}
+                                </td>
+
+                            </tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </section>
+</div>
+
+<div>
+    <section class="container">
+        <div class="row">
+            <div class="col-12">
+                <h4>Demographics</h4>
+                <div class="table-responsive">
+                    <table class="table table-striped comparison-table">
+                        <thead class="thead-dark">
+                            <tr>
+                                <th>Candidate</th>
+                                <th>Age</th>
+                                <th>Voter Reg</th>
+                                <th>Home Value</th>
+                                <th>Area</th>
+                                <th>Education</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for candidate in candidates %}
+                            <tr>
+                                <td>
+                                    <a href="{% url "candidate_detail" candidate.slug %}">{{ candidate.fullname }}</a>
+                                    <p>
+                                        <i>
+                                            {% if candidate.pronoun == 'he' %}
+                                                He/Him
+                                            {% elif  candidate.pronoun == 'she'%}
+                                                She/Her
+                                            {% elif  candidate.pronoun == 'they'%}
+                                                They/Them
+                                            {% endif %}
+                                        </i>
+                                    </p>
+                                </td>
+
+                                <td>
+                                    {{ candidate.date_of_birth|timesince }}
+                                </td>
+
+                                <td>
+                                    {% if candidate.date_of_registration %}
+                                        {{ candidate.date_of_registration }}
+                                    {% endif %}
```

Comment:
I think you probably don't need the ifs around these blocks. If the value is None, Django won't show anything
```suggestion
                                        {{ candidate.date_of_registration }}
```

Reply:
django was showing the text `None` which looked bad :(